student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Student < Person
+  def initialize(classroom)
```

Comment:
- [ ] I will implore you to add other parameters too since you have them in the super.